apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/cql/builder/QueryBuilderImpl.java

```diff
@@ -1336,10 +1329,13 @@ private String updateQuery() {
   }
 
   private void appendWheres(StringBuilder builder) {
+    // JSON API fully rely on Expression<BuildCondition> instead of List<BuildCondition>
     if (this.whereExpression != null) {
       appendConditions(this.whereExpression, " WHERE ", builder);
       return;
     }
+
+    // keep wheres, since Docs API still use it, although in deprecate plan
```

Comment:
Just include "keep wheres since Docs API still uses it", no need to mention deprecation here.

Reply:
solved